src/mam4xx/drydep.hpp

```diff
@@ -1017,9 +1018,9 @@ void aero_model_drydep(
   const int max_species = static_cast<int>(AeroId::None);
   Kokkos::parallel_for(
       Kokkos::TeamThreadRange(team, ntot_amode * (1 + max_species)),
-      KOKKOS_LAMBDA(int kk) {
-        const int imode = kk / (1 + max_species);
-        const int lspec = kk % (1 + max_species) - 1;
+      KOKKOS_LAMBDA(int ic) {
```

Comment:
Following Conrad's recommendation, we should use `TeamVectorRange` instead of `TeamThreadRange` and switch `KOKKOS_LAMBDA` to `[&]`.

Reply:
Done